daemon/snapper/snapper.go

```diff
@@ -46,6 +52,8 @@ type args struct {
 	interval       time.Duration
 	fsf            *filters.DatasetMapFilter
 	snapshotsTaken chan<- struct{}
+	hooks          config.HookList
+	dryRun         bool
```

Comment:
`dryRun` is dead (always false) => I wouldn't even add it at this point

Reply:
So put in in the HookEnv as a `false` literal for now, per your earlier comment that you'd like it to be set in the hook process environment even if it's not currently used?

Reply:
Yup. I'm currently still undecided whether we want it to be

1. `false`, `true` => `if [ "$ZREPL_DRYRUN" = "false" ]; then X; else Y; fi` 
2. "ZREPL_DRYRUN=", "ZREPL_DRYRUN=true" => `if [ -z "$ZREPL_DRYRUN" ]; ...`

The second variant is a little more typical.

Reply:
I support the latter, as it's more shell-like.